examples/css/index.html

```diff
@@ -26,22 +26,152 @@ <h1 class="mistica-text-6">Hello mistica css</h1>
+                        <p class="mistica-card__subtitle">Subtitle</p>
+                        <p class="mistica-card__description">Description</p>
+                        <div class="mistica-card__buttons">
+                            <button class="mistica-button-primary-small">Action</button>
+                            <button class="mistica-button-link-small">Link</button>
+                        </div>
+                    </section>
+                    <section class="mistica-card">
+                        <div class="mistica-tag-promo">Headline</div>
+                        <h2 class="mistica-card__pretitle">Pretitle</h2>
+                        <h2 class="mistica-card__title">Title</h2>
+                        <p class="mistica-card__subtitle">Subtitle</p>
+                        <p class="mistica-card__description">Description</p>
+                        <div class="mistica-card__buttons">
+                            <button class="mistica-button-primary-small">Action</button>
+                            <button class="mistica-button-link-small">Link</button>
+                        </div>
+                    </section>
+                    <section class="mistica-snap-card">
+                        <h2 class="mistica-card__title">Title</h2>
+                        <p class="mistica-card__subtitle">Subtitle</p>
+                        <p class="mistica-card__description">Description</p>
+                    </section>
+                </div>
+                <h4 class="mistica-text-title1">NakedCard / SmallNakedCard</h4>
+                <div class="cards-group">
+                    <div class="mistica-naked-card">
+                        <img class="mistica-card__media" src="https://picsum.photos/1200/1200" />
+                        <div class="mistica-tag-promo">Headline</div>
+                        <h2 class="mistica-card__pretitle">Pretitle</h2>
+                        <h2 class="mistica-card__title">Title</h2>
+                        <p class="mistica-card__subtitle">Subtitle</p>
+                        <p class="mistica-card__description">Description</p>
+                        <div class="mistica-card__buttons">
+                            <button class="mistica-button-primary-small">Action</button>
+                            <button class="mistica-button-link-small">Link</button>
+                        </div>
+                    </div>
+                    <div class="mistica-small-naked-card">
+                        <img class="mistica-card__media" src="https://picsum.photos/1200/1200" />
+                        <h2 class="mistica-card__title">Title</h2>
+                        <p class="mistica-card__subtitle">Subtitle</p>
+                        <p class="mistica-card__description">Description</p>
+                    </div>
+                </div>
+
+                <h4 class="mistica-text-title1">DisplayMediaCard / DisplayDataCard</h4>
```

Comment:
DisplayMediaCard is showing a border, [but it shouldn't](https://mistica-web.vercel.app/playroom/#?code=N4Igxg9gJgpiBcIA8ARAlgZwA4BsCGAngLIxRp4DCeATlADoB2ABEwBYx5Q5oMwC8wJABU8AcyYAXAln50QWahAC2EOQD4AEhy48YSAPQjRagL6MWCmBLQScskAAVqVm3bnnJr%2B0K-vmTWAwwajQsawgGPjkUGCCQsLQIvxYAIzwwAGtRRQBXBigASSUxe1YJCSwMeH19LDQwDBylADosVggJCAx9AEYAJgAGAd7BgeSmFJzyiIEPFiQAISnOhic0YuoCJgxinBw2ZwAzKJAyiqqa0QgIUTtmyCV1OZYmAEEwcIZngyXp1ZCNgQ1B4zP4IhQcF1%2BMAABQASiYfDUTGAJlBLHSnwwAgA2s9gM8WAVIAx4ExiREADJoURlBg8UQAJRgohy%2BGoABpCUwIk5YlUmPDEcjUVz-C8mPgUjAcGS5NTaRJ6QxRHIxRKTOqXgTxS8wOxMqQnBBKmSdRKJRTSeSSQBlCQ0ABiaD2pC1FsleGlsqYcn1MEN9BA7pemu5eX9geNppR3KJJLJVvtNGZrPZIYlUplcpAEYNGVIau5Yd1LFghzwbIkFHzpDJFZwGBgGZ5DBreBVMDJQqRKJLGq1AF1QfpgQxgyAJOwlLEEDiQEQIAA3TAO6ggQcckAAdzQUCnGDnAHYAGwADg5PQAzOfh0A)

Reply:
should be fixed